gulpfile.js

```diff
@@ -0,0 +1,38 @@
+const gulp = require("gulp");
```

Comment:
This part is a bit confusing to me and I don't know; but why are we removing `webpack_loader` and switching to using gulp instead?

Reply:
I can say a little about that, though I'm happy to discuss the options before moving ahead.
This came about as a result of trying to upgrade webpack from v4 to v5, which caused things to break. In a separate conversation with Jeremy, I mentioned that `webpack_loader` seemed to not work with webpack v5 when starting a new project with jade-truffle, and he mentioned that SC had decided to stop using `webpack_loader` because they didn't need it, and have [an open pull request](https://github.com/caktus/jade-truffle/pull/32) to update jade-truffle to remove it, besides other changes.
Due to these things, I encouraged replacing `webpack_loader` in this project with the gulp approach from the jade-truffle pull request.

Let us know if you have concerns, or if keeping `webpack_loader` is better.

Reply:
Thanks for clarifying @dchukhin and that's good to know.